lienol/luci-app-passwall/luasrc/controller/passwall.lua

```diff
@@ -172,7 +172,7 @@ function connect_status()
     local start_time = os.time()
     if luci.http.formvalue("type") == "google" then
         e.status = luci.sys.call(
-                       "echo $(/usr/share/passwall/test.sh test_url 'www.google.com') | grep 200 >/dev/null") ==
+                       "echo $(/usr/share/passwall/test.sh test_url 'www.google.com/generate_204') | grep 200 >/dev/null") ==
```

Comment:
generate_204 | grep 200?

Reply:
> generate_204 | grep 200?

It use wget instead of curl,when the result wasn't empty,and status=200,you can see the code in the test.sh.It does cause some confusion.